src/notification.c

```diff
@@ -365,25 +381,10 @@ void notification_init(struct notification *n)
         rule_apply_all(n);
 
         /* UPDATE derived fields */
-        notification_update_icon(n);
         notification_extract_urls(n);
         notification_format_message(n);
 }
 
-static void notification_update_icon(struct notification *n)
-{
-        g_return_if_fail(n);
-
-        g_clear_object(&n->icon);
-
-        if (n->raw_icon)
-                n->icon = get_pixbuf_from_raw_image(n->raw_icon);
-        else if (n->iconname)
-                n->icon = get_pixbuf_from_icon(n->iconname);
-
-        n->icon = icon_pixbuf_scale(n->icon);
```

Comment:
Uhm, where did the icon scaling logic go?

Reply:
Good point. I deleted this right before opening this PR by accident. :see_no_evil: